apps/api/src/app/events/services/chat-service/handlers/base.handler.ts

```diff
@@ -15,7 +15,7 @@ export abstract class BaseChatHandler implements IChatHandler {
 
   async send(chatContent: IChatOptions) {
     if (process.env.NODE_ENV === 'test') {
-      return null;
+      return {};
```

Comment:
I particularly do not like this. Returning an empty object I think it can cause a lot of problems and doesn't make much sense, to be honest.